service-base/src/main/java/org/eclipse/hono/service/plan/PrometheusBasedResourceLimitChecks.java

```diff
@@ -118,11 +121,14 @@ public void setPort(final int port) {
         }
     }
 
-    private Future<Long> queryForSumInPrometheus(final String metricName, final TenantObject tenant) {
+    @PostConstruct
+    private void init() {
+        queryUrl = String.format("http://%s:%s/api/v1/query", getHost(), getPort());
+    }
+
+    private Future<Long> executeQuery(final String queryParams) {
```

Comment:
My understanding of the code is that the parameter *is* the actual query, not  *queryParams*, isn't it?.

Reply:
From the perspective of _PromQL_, it is query. In connection to the _httpRequest_ it is _queryParams_. I think in the above case, _query_ suits better than the _queryParams_. I will update it in one of the future PRs for _resource-limits_.